docs/spec/WIP_slashing/transactions.md

```diff
@@ -0,0 +1,19 @@
+
+### TxProveLive
```

Comment:
This is now "MsgUnrevoke".

What other transactions will be included in this file, if any?

Reply:
WIP, another PR